packages/graphql-server/src/resolvers/contact/__tests__/resolvers.test.ts

```diff
@@ -1,15 +1,17 @@
-import { queryContact, queryContacts, createContact } from '../resolvers'
-import { GetContactByIdArgs, GetContactsArgs } from '../contact'
+import { queryContact, queryContacts, createContact, updateContact } from '../resolvers'
+import { GetContactByIdArgs, GetContactsArgs } from '../types'
 import { mockContext } from '../../../__mocks__/context'
 import { contact } from '../__mocks__/contact'
 import { contacts } from '../__mocks__/contacts'
 import { mockCreateArgs } from '../__mocks__/create-args'
+import { mockUpdateArgs } from '../__mocks__/update-args'
 import errors from '../../../errors'
 
 jest.mock('../services', () => ({
   getContactById: jest.fn(() => contact),
   getContacts: jest.fn(() => contacts),
-  createContact: jest.fn(() => contact),
+  createContact: jest.fn(() => true),
+  updateContact: jest.fn(() => contact),
 }))
```

Comment:
Can you add mocks for `error` and `logger` so it won't  print unnecessary log like this when running tests
<img width="1282" alt="Screen Shot 2020-03-02 at 14 28 06" src="https://user-images.githubusercontent.com/26746668/75654330-137a8a80-5c92-11ea-9d9d-bff6d1e58f8c.png">